Reply/app/src/main/java/com/materialstudies/reply/ui/compose/ComposeFragment.kt

```diff
@@ -69,13 +89,24 @@ class ComposeFragment : Fragment() {
             email = composeEmail
 
             composeEmail.nonUserAccountRecipients.forEach { addRecipientChip(it) }
+            composeEmail.nonUserAccountRecipients.forEach{  addRecipientsCard(it) }
+
+            binding.handlers = containerTranformHandler
```

Comment:
i think we should discuss whether we need the `ContainerTransformHandler`. it feels sort of confusing to me that some of the `containerTransform` method calls are hooked up here and some are hooked up via data binding. it might be clearer if we hook up all of the `containerTransform` calls in Kotlin in the same place if possible.